build.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencies {
     // https://chat.fhir.org/#narrow/stream/179166-implementers/topic/New.20validator.20JAR.20location
     // the ig-publisher uses this one too
     // https://github.com/HL7/fhir-ig-publisher/blob/master/pom.xml#L68
-    implementation("ca.uhn.hapi.fhir", "org.hl7.fhir.validation", "5.3.14")
+    implementation("ca.uhn.hapi.fhir", "org.hl7.fhir.validation", "5.5.10")
```

Comment:
Latest validation version is now "5.6.21". I tested it locally and it didn't break anything.

Reply:
This will be fixed in PR #43 So I think this is fine.

Reply:
Sounds good. I'll approve and merge.